Update advisorengine attribute with insights-iop attribute (#4085)
theforeman/foreman-documentation: guides/common/modules/con_installing-and-configuring-insights-iop.adoc

:_mod-docs-content-type: CONCEPT

[id="installing-and-configuring-{insights-iop-id}"]
= Installing and configuring {insights-iop}

{insights-iop} analyzes system health and configuration by applying predefined rules to a small set of local data, such as installed packages, running services, and configuration settings.
When you install {insights-iop} locally, you can generate {Insights} recommendations without sending system data to Red{nbsp}Hat services.

[IMPORTANT]
====
With {insights-iop} enabled, you cannot use the hosted {Insights} services for hosts registered to your {Project}.

> Enabling {insights-iop} prevents you from using any Red{nbsp}Hat Hybrid Cloud Console services on the registered hosts.

Reviewer: ```suggestion
Enabling {insights-iop} prevents you from using any {RHCloud} services on hosts registered to {Project}.
```
We have an attribute for the RH HCC - please double check this is the correct attribute name.
And the hosts can be registered to HCC as well AFAIK, so let's be a bit clearer.

> [id="installing-and-configuring-insights-iop"]

Author: You're right. Should I add an attribute in this PR for the anchor?

> [id="installing-and-configuring-insights-iop"]

Reviewer: Yes, please.

= Installing and configuring {insights-iop}

:FeatureName: {insights-iop}

> include::snip_technology-preview.adoc[]

Author: I will remove the snippet for the TP since advisor is planning on going GA in 6.18.

{insights-iop} analyzes system health and configuration by applying predefined rules to a small set of local data, such as installed packages, running services, and configuration settings.

> When you install {insights-iop} locally, you can generate {Insights} recommendations without sending system data to Red{nbsp}Hat services.

Reviewer: Should we mention the vulnerability stuff as well?

Author: @Lennonka I don't know if the vulnerability needs to be mentioned here since you are addressing it in #3938 . @chris1984 said it can be addressed in Managing hosts in that PR by you.
